conf/app.item.dangerousGoods.index.routes

```diff
@@ -0,0 +1,5 @@
+
+GET        /:lrn/items/:itemIndex/dangerousGoods/:dangerousGoodsIndex/un-number                     controllers.item.dangerousGoods.index.UNNumberController.onPageLoad(lrn: LocalReferenceNumber, mode: Mode = NormalMode, itemIndex: Index, dangerousGoodsIndex: Index)
```

Comment:
Could you change dangerousGoods to dangerous-goods? I only spotted when I pulled the branch down